src/meson.build

```diff
@@ -1,7 +1,9 @@
 prog_python = find_program('python3')
 
+vulkan_prefix = meson.get_external_property('sys_root', '') + vulkan_dep.get_variable('prefix')
+
 vk_xml = join_paths([
-    vulkan_dep.get_variable('prefix'),
+    vulkan_prefix,
```

Comment:
Could we add `meson.get_external_property('sys_root', '')` to the join_paths list instead?